ffv1.md

```diff
@@ -472,6 +472,52 @@ SVGI:!---
 SVGC:rangebinaryvalues7.svg=$$j_{0}=2$$
 AART:j_(0) = 2
 
+```c
+    range = 0xFF00;
+    end   = 0;
+    low   = get_bits(16);
```

Comment:
In theory, couldn't the bitstream be 1-byte long?
In my implementation of the range coder decoding I have something more like:

```c
    range = 0xFF;
    end   = 0;
    low   = get_bits(8);
    refill();
```

(with refill() moved above this text)
The reference decoder does not handle a 1-byte `Slice` and in practice the specification currently prevents a so small `Slice` due to the count of fields in it, but in the future it could be the case.
Maybe we would update the spec at the same time as the reference decoder if a 1-byte `Slice` is possible in practice...

Reply:
I don't really have a strong opinion on the matter.